cleanup

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+rm -rf	autom4te.cache/ config.* src/Makevars \
+    	src/*.o src/*.d src/*.a src/*.dll src/*.so \
+    	*/*~ *~ 
+
+
```

Comment:
You should avoid the trailing lines here. Also, scripts need to be `chmod +x` before you commit them (git tracks permissions as well). You can add both in as a fixup.